tutorials/model_exploration.md

```diff
@@ -0,0 +1,75 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Model Exploration
+
+```{note}
+The purpose of this tutorial is to explore a model by learning the following:
+1. Parsing (loading) an existing model
+2. Querying an existing model.
+```
+
+For this tutorial, we'll use an existing equipment model of a variable air volume (VAV) terminal unit with cooling only from section 4.1 of ASHRAE Guideline 36-2021. This and other example models are available from [Open223 Models](open223-resources).
+
+## Model Parsing
+
+First, we'll create a new empty graph then parse (load) an existing graph into it using the Python RDFLib library.
+
+```{code-cell}
+from rdflib import Graph
+
+# Create a Graph
+g = Graph()
+
+# Parse in an RDF file hosted on the Internet
+g.parse("https://raw.githubusercontent.com/open223/models.open223.info/main/models/guideline36-2021-4-1.ttl", format="ttl")
```

Comment:
You can probably replace the URL with 'https://models.open223.info/guideline36-2021-4-1.ttl' now. I tested this locally and it works fine